README.md

```diff
@@ -2,6 +2,8 @@
 
 Visualization tools for CoDyCo Y4 Demo
 
+Instruction for using testbot_description ros package can be found in https://github.com/robotology-playground/codyco-y4-viz/testbot_description/README.md file.
+
```

Comment:
For links to internal documents in Markdown Github files, you can use relative links : https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files .